intel-retail/automated-self-checkout
# feat: Add OVMS client script into go main configuration

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- build ovms client: `make build-ovms-client`
- start up camera simulator if any
- run docker run command like `sudo ./docker-run.sh --platform core --inputsrc http://127.0.0.1:8554/camera_0 --workload opencv-ovms `
- exam docker logs: `docker logs ovms-client0`  and it should looks no error
- exam the result log in results folder: `cat ./result/maskrcnn.log`  should be no error

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
